app/views/pages/home.html.erb

```diff
@@ -1,5 +1,6 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="container">
+  <h1>Homepage</h1>
 
-<p>signed in? : <%= user_signed_in? %></p>
-<p>email: <%= current_user.email if user_signed_in? %></p>
+  <p><strong>User Signed in?</strong> : <%= user_signed_in? %></p>
+  <p>email: <%= current_user.email if user_signed_in? %></p>
```

Comment:
@rx-chris do you think I should add `strong` here too?

Reply:
Yes